1.8/translations/translations.md

```diff
@@ -29,3 +29,17 @@ If these exist, they should be uploaded to `images/<language name>/` and `groupi
 #### Finishing Touches
 
 Go to **User CP > Edit Options**. You should see the translation in the **Other Options** menu. Try out the new translation and make sure that everything works.
+
+### Default forum translation
+
+If you want your new translation to be default for all new users and guests you can change it by going to **Admin CP > Configuration > Settings > General Configuration > Default Language**.
+This would make this the default translation for all new users and guests.
+
+**Please note:** Changing this in Admin CP will not force new default translation to existing users on board. They need to change it manually in their profiles.
```

Comment:
this should be:
**Please note:** Changing this in the Admin CP will not force a new default translation for existing users on the forum. They will need to change it manually in their profiles.